app/pages/lab/workflow.jsx

```diff
@@ -0,0 +1,1042 @@
+              <p className="form-help"><small>Status indicates whether a workflow is active or inactive. Active workflows are available to volunteers and classifications count toward subject retirement. Workflow status can be managed under the Visibility section within the Project Builder.</small></p>
+            </div>
+
+            <hr />
+
+            <div>
+              <span className="form-label">Associated subject sets</span><br />
+              <small className="form-help">Choose the set of subjects you want to use for this workflow.</small>
+              <SubjectSetLinker
+                project={this.props.project}
+                workflow={this.props.workflow}
+              />
+            </div>
+
+            <hr />
+
+            <div>
+              <AutoSave resource={this.props.workflow}>
+              <span className="form-label">Set annotation persistence</span><br />
+              <small className="form-help">Save the annotation of the task you are on when the back button is clicked.</small>
+              <br />
+              <label>
+                <input ref="persistAnnotation" type="checkbox" checked={this.props.workflow.configuration.persist_annotations} onChange={this.handlePersistAnnotationsToggle.bind(this)} />
+                Persist annotations
+              </label>
+              </AutoSave>
+              <hr />
+            </div>
+
+            <div className={disabledIfWorkflowInactive}>
+              <AutoSave resource={this.props.project}>
+                <span className="form-label">Set as default</span><br />
+                <small className="form-help">If you have more than one workflow, you can set which should be default. Only one can be default.</small>
+                {projectLiveWorkflowInactive ? <span><br /><small className="form-help">Inactive workflows on live projects cannot be made default.</small></span> : undefined}
+                <br />
+                <label>
+                  <input ref="defaultWorkflow" type="checkbox" disabled={projectLiveWorkflowInactive} checked={this.props.project.configuration?.default_workflow === this.props.workflow.id} onChange={this.handleDefaultWorkflowToggle.bind(this)} />
+                  Default workflow
+                </label>
+              </AutoSave>
+            </div>
+
+            <hr />
+
+            <div ref="link-tutorials-section">
+              <span className="form-label">Associated tutorial {Array.from(this.props.project.experimental_tools).includes('mini-course') ? "and/or mini-course" : undefined}</span><br />
+              <small className="form-help">Choose the tutorial {Array.from(this.props.project.experimental_tools).includes('mini-course') ? "and/or mini-course" : undefined} you want to use for this workflow.</small><br />
+              <small className="form-help">Only one can be associated with a workflow at a time.</small>
+              <div>
+                {this.renderTutorials()}
```

Comment:
Rendering the return value of a function like this means that `renderTutorials` can probably be extracted into a component.

Reply:
Moved into #6228.